engine/src/main/resources/templates/Advisor/dailyEmailBodyV2.html

```diff
@@ -0,0 +1,156 @@
+                {#case "4"}<img src="https://console.redhat.com/apps/frontend-assets/email-assets/img_critical.png" alt="Critical severity" width="70" border="0">
+              {/switch}
+            </td>
+            <td style="width: 20%">{action.context.advisor.resolved_recommendations.get(rec).systems}</td>
+          </tr>
+        {/for}
+        </tbody>
+      </table>
+      {#if action.context.advisor.deactivated_recommendations.orEmpty.size > 0}
+        <div style="height: 24px"></div>
+      {/if}
+{/if}
+{#if action.context.advisor.deactivated_recommendations.orEmpty.size > 0}
+    <table class="rh-data-table rh-m-bordered">
+       <thead>
+        <tr>
+          <th style="width: 60%; padding-left: 26px;">&nbsp;Deactivated Recommendation{#if action.context.advisor.deactivated_recommendations.orEmpty.size > 1}s{/if}</th>
+          <th style="width: 40%">Total Risk</th>
+        </tr>
+        </thead>
+        <tbody>
+        {#for rec in action.context.advisor.deactivated_recommendations.keySet()}
+          <tr>
+            <td style="width: 60%; word-wrap: anywhere;">
+                {#let isIncident = action.context.advisor.deactivated_recommendations.get(rec).has_incident.or("false")}
+                    <svg class="fa-exclamation-circle" height="16" width="16" viewBox="0 0 512 512" aria-hidden="true" role="img" {#if !isIncident || (true != isIncident && "true" != isIncident)} style='visibility: hidden'{/if}>
+                        <path><title>Incident</title></path>
+                    </svg>
+                {/let}
+                <a target="_blank" href="{environment.url}/insights/advisor/recommendations/{rec}">{action.context.advisor.deactivated_recommendations.get(rec).rule_description}</a>
+                {#if action.context.advisor.deactivated_recommendations.get(rec).has_incident && true == action.context.advisor.deactivated_recommendations.get(rec).has_incident}
+                    <span class="rh-incident">Incident</span>
+                {/if}
+            </td>
+            <td style="width: 40%">
+              {#switch action.context.advisor.deactivated_recommendations.get(rec).total_risk}
+                {#case "1"}<img src="https://console.redhat.com/apps/frontend-assets/email-assets/img_low.png" alt="Low severity" width="55" border="0">
+                {#case "2"}<img src="https://console.redhat.com/apps/frontend-assets/email-assets/img_moderate.png" alt="Moderate severity" width="85" border="0">
+                {#case "3"}<img src="https://console.redhat.com/apps/frontend-assets/email-assets/img_important.png" alt="Important severity" width="86" border="0">
+                {#case "4"}<img src="https://console.redhat.com/apps/frontend-assets/email-assets/img_critical.png" alt="Critical severity" width="70" border="0">
+              {/switch}
+            </td>
+          </tr>
+        {/for}
+        </tbody>
+    </table>
+{/if}
+{/content-body-section1}
+{#content-body-section2}
```

Comment:
Section 2 has no title, is that on purpose?

Reply:
Yes it is